src/test/java/uk/gov/justice/digital/hmpps/csr/api/service/DetailServiceTest.kt

```diff
@@ -50,24 +50,35 @@ internal class DetailServiceTest {
     @DisplayName("Get Staff Details")
     inner class DetailServiceTests {
 
+        @Test
+        fun `Should pad 'from' dates by -1 days`() {
+            val details = listOf(getValidShiftDetail(123L, 456L))
+            every { sqlRepository.getDetails(from.minusDays(1), to, quantumId) } returns details
```

Comment:
It'll be a little easier to read if this is set as a test suite scope variable such as

```
val paddedFrom = shiftDate.minusDays(2)
```

Reply:
👍